src/components/Button/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './Button.module.css';
+
+const Button = () => {
```

Comment:
Nazwa Button sugeruje trochę, że ten Button jest uniwersalny. Jeśli ktoś się nie wczyta nie wie, że to button, który przesyła Cię na konkretną stronę i ma określoną nazwę. 
Rozważ zmianę nazwy na mówiącą trochę więcej ;) np. AddNewButton?

Reply:
Z opóźnieniem też widzę, że on nie jest nigdzie wykorzystywany. Uważam, że całkiem sensownie go zastąpiłaś w PicturesCollection.js.